app/scripts/translations/en.js

```diff
@@ -588,7 +588,7 @@ x_MetaMask                      :  'MetaMask / Mist',
 x_Mnemonic                      :  'Mnemonic Phrase',
 x_ParityPhrase                  :  'Parity Phrase',
 x_Password                      :  'Password',
-x_PasswordDesc                  :  'This password * encrypts * your private key. This does not act as a seed to generate your keys. **You will need this password + your private key to unlock your wallet.**',
+x_PasswordDesc                  :  'This password * encrypts * your private key. This does not act as a seed to generate your keys. **You will need this password + your keystore file to unlock your wallet.**',
```

Comment:
should we change `private key` on this line as well?

Reply:
Not quite sure what you're referencing. 

The password encrypts the PK in the keystore file, so you need keystore + password to get the private key back out.  Saying "You will need this password + your private key" implies you already have your private key, in which case there's no need for either the password or keystore file.